### PR TITLE
fix: return status from remove job and start job

### LINKED ISSUE #753: fix(jobplugin): improve feedback from jobplugin
- [x] Show status of job at all times
- [x] Format the logs (not one single line)
- [x] hide buttons in recipe
- [x] show all logs in widget
- [x] progress bar
- [x] possibility to post status, progress and log to job-api from the job
- [x] round green button to start, when running color should change to red to stop
- [x] pop up confirmation on stop job
- [x] remove/stop jobs

### PR DESCRIPTION
**Pull request by collinlokken**
## What does this pull request change?
Return status from job api when job is registered and stopped/deleted

## Why is this pull request needed?
Allows the frontend (job plugin) to change state based on responses from the api instead of guessing what has happended.

## Issues related to this change
PR in job plugin: https://github.com/equinor/dm-core-packages/pull/814

### CODE REVIEW

File: src/job_handler_plugins/local_container/__init__.py [hunk 1]
```diff
@@ -62,30 +62,18 @@ def start(self) -> str:
...
-                status = JobStatus.RUNNING
-            elif container.attrs["State"]["ExitCode"] >= 1:
-                status = JobStatus.FAILED
-            elif container.attrs["State"]["ExitCode"] == 0:
-                status = JobStatus.COMPLETED
-            return status, None, self.job.percentage
-        except docker.errors.NotFound as error:
-            logger.error(f"Failed to poll progress of local container: {error}")
-            return JobStatus.UNKNOWN, f"Failed to poll progress of local container: {error}", self.job.percentage
+        return self.job.status, self.job.log, self.job.percentage
```
Comment (soofstad) [inline src/job_handler_plugins/local_container/__init__.py]: Why remove the old status stuff?

File: src/job_handler_plugins/radix/__init__.py [hunk 2]
```diff
@@ -60,20 +60,4 @@ def remove(self) -> Tuple[str, str]:
...
-        job_status = JobStatus.UNKNOWN
-        match (response_json["status"]):  # noqa
-            case "Running":  # noqa
-                job_status = JobStatus.RUNNING
-            case "Failed":  # noqa
-                job_status = JobStatus.FAILED
-            case "Succeeded":  # noqa
-                job_status = JobStatus.COMPLETED
-        return job_status, None, self.job.percentage
+        return self.job.status, self.job.log, self.job.percentage
```
Comment (soofstad) [inline src/job_handler_plugins/radix/__init__.py]: Why remove?

**Review by soofstad — CHANGES_REQUESTED** [review]

File: src/job_handler_plugins/radix/__init__.py (hunk 2 above)
Comment (collinlokken) [reply src/job_handler_plugins/radix/__init__.py]: discussed

File: src/job_handler_plugins/local_container/__init__.py (hunk 1 above)
Comment (collinlokken) [reply src/job_handler_plugins/local_container/__init__.py]: discussed

File: src/job_handler_plugins/local_container/__init__.py [hunk 3]
```diff
@@ -58,34 +57,34 @@ def start(self) -> str:
...
-            status = self.job.status
             if container.status == "running":
-                status = JobStatus.RUNNING
-            elif container.attrs["State"]["ExitCode"] >= 1:
-                status = JobStatus.FAILED
+                return JobStatus.RUNNING, "Job is running", 0
             elif container.attrs["State"]["ExitCode"] == 0:
-                status = JobStatus.COMPLETED
-            return status, None, self.job.percentage
+                return JobStatus.COMPLETED, "Local container completed successfully", 1
```
Comment (soofstad) [inline src/job_handler_plugins/local_container/__init__.py:79]: Shouldn't progress be 100% here? not 1

File: src/default_job_handlers/recurring_job/__init__.py [hunk 4]
```diff
@@ -26,20 +27,19 @@ def __init__(
     def start(self) -> str:
         msg = f'Starting scheduled job from "{self.job.dmss_id}"'
         logger.info(msg)
-        self.job.append_log(msg)
+        update_progress(self.job.job_uid, progress=Progress(logs=msg, percentage=0), overwrite_log=False)
```
Comment (soofstad) [inline src/default_job_handlers/recurring_job/__init__.py]: hjg

**Review by soofstad — CHANGES_REQUESTED** [review]

File: src/job_handler_plugins/local_container/__init__.py (hunk 3 above)
Comment (collinlokken) [reply src/job_handler_plugins/local_container/__init__.py:79]: no

**Review by soofstad — APPROVED** [review]

**Pull request merged**